pages/docs/configuration/configuration-mapping.md

```diff
@@ -98,6 +98,26 @@ The interpolation problem might not be well-defined if you map along an axis-sym
 All data mappings are executed during `advance` and not in `readBlockVectorData` etc., cf. the section on  [how to couple your own code](couple-your-code-overview.html).
 {% endnote %}
 
+## Geometric multiscale mapping
+
+Geometric multiscale mapping enables the coupling of dimensionally heterogeneous coupling participants. This can be helpful if some regions of the domain are of less interest and can therefore be treated with simplified, lower dimensional models. One has to differentiate between axial and radial geometric multiscale mapping, i.e. mapping values at an interface boundary surface between two models for the former or coupling along an intersection of the models without such an interface boundary surface. 
+
+Currently, only axial geometric multiscale coupling is supported.
+
+<img src="https://user-images.githubusercontent.com/99761626/189532277-e39a1075-c479-4412-8e06-5c5e2ad71642.png" alt="Axial geometric multiscale mapping" width="500"/> <img src="https://user-images.githubusercontent.com/99761626/191258600-84a7de22-b056-4d5c-b01a-85cfae4566ff.png" alt="Radial geometric multiscale mapping" width="500"/>
```

Comment:
Move the image here (e.g., to https://github.com/precice/precice.github.io/tree/master/images/docs/configuration) and name it accordingly (e.g., `doc-mapping-geometric-multiscale.png`). Also provide the source file of the image in the same folder.

Note that the `alt=` is only mentioning axial mapping, while the picture also shows radial.

Reply:
I moved the images into the repository. They don't render in the preview here, but I have checked locally.